_posts/2015-07-25-bear-foraging.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: It's hot out there for a bear
+author: neuroamanda
+---
+
+_Foraging for food takes precedence over staying cool_
+
+Grizzly bears have a dilemma.
+They only have about seven months a year to eat 12 months' worth of food before they go into hibernation.
+This imperative to eat is at odds with high temperatures during the best feeding months.
+Bears don't have sweat glands, yet they must develop thick under-fur and a layer of fat for the winter.
+Do the bears reduce their activity to beat the heat, or do they stick to their eating mission?
+
+Wildlife researchers followed small groups of grizzlies in British Columbia and recorded their activities.
+Previous research has found that bears reduce their activity levels when temperatures reached 20&deg;C (68&deg;F).
```

Comment:
No &deg;F? Who uses it, seriously? ;)

Also, I think it should be "20 &deg;C", i.e. with a space.